src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.scss

```diff
@@ -5,6 +5,24 @@
     position: absolute;
   }
 
+  &__background-image {
+    width: 100%;
+  }
+
+  &__background-image-placeholder {
+    width: 100%;
+    height: 1000px;
```

Comment:
Sounds like a bad idea to have this hardcoded. Can you use 100% or 100vh instead?

Reply:
Initial intention was to set to 1200px (i mistakenly used 1000) as in text: *Recommended size: 2000px / 1200px*. But yeah, `vh` is better solution. Fix in https://github.com/sparkletown/sparkle/commit/f033a214d20db26a1e8d035a115ccd345272d54f